cmd/gzv/cli/miner_rpc.go

```diff
@@ -72,13 +72,14 @@ func startHTTP(endpoint string, apis []rpc.API, modules []string, cors []string,
 	if endpoint == "" {
 		return nil
 	}
+	isPruneMode := core.BlockChainImpl.IsPruneMode()
```

Comment:
we can get the prune mode config in current package directly. do not need to add this dependence for core package